src/Tally.java

```diff
@@ -0,0 +1,34 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+public class Tally {
+    public static void main(String[] args) {
+        List<String> listOfVotes = new ArrayList<>();
+        for (String currentString : args){
+            listOfVotes.add(currentString);
+        }
+        System.out.println(tally(listOfVotes));
+    }
+
+    public static String tally(List<String> input){
+        HashMap<String, Integer> tallyMap = new HashMap<>();
+        int maxVotes = 0;
+        String returnedCandidate = "ERR: no votes?";
+        for(String currentString : input){
+            if(!tallyMap.containsKey(currentString)){
+                tallyMap.put(currentString, 1);
+            }else{
+                tallyMap.put(currentString, tallyMap.get(currentString) + 1);
+            }
```

Comment:
For readability, please space out your code as you concatenate:
`returnedCandidate = "Run off between " + returnedCandidate + " and " + currentString;`